pojo/src/main/java/com/spotify/hamcrest/pojo/IsPojo.java

```diff
@@ -64,7 +64,7 @@
     return where(
         methodName,
         self -> {
-          final Method method = self.getClass().getDeclaredMethod(methodName);
```

Comment:
This technically makes us no longer support private/package private methods; add test for that?

Reply:
Fixed